### PR TITLE
feat(structure): update multiple_slice_cat

### PR DESCRIPTION
**Pull request by jyjyjyjyjyjyjyj**

### CODE REVIEW

File: xpu_graph/passes/patterns/structure/fuse_multiple_slice_cat.py [hunk 1]
```diff
@@ -13,52 +14,89 @@ def fuse_multiple_cat(graph_module: fx.GraphModule):
...
         graph_module.add_submodule(
-            "fuse_slice_cat",
+            "fuse_slice_cat_v2",
             self.target_mod(),
         )
 
         # merge multiple "fuse_slice_cat" with the same src_node to one "fuse_slice_cat"
-        changed = changed | fuse_multiple_cat(graph_module)
+        # fused_slice_cat(x, [A, B]) + fused_slice_cat(x, [C, D]) ->fused_slice_cat_v2(x, [[A, B], [C, D]])
+        changed = changed | fuse_multiple_cat1(graph_module)
```
Comment (zhangjihang-BD) [inline xpu_graph/passes/patterns/structure/fuse_multiple_slice_cat.py:99]: 这个写法对吗，如果changed==True，会直接bypass 后面表达式的pattern呢

File: xpu_graph/passes/patterns/targets/mlu/triton_kernel/fused_slice_v2.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                        strides=(slice_len, 1),
+                        offsets=(l * BLOCK_SIZE_R, l_c * BLOCK_SIZE_C),
+                        block_shape=(BLOCK_SIZE_R, BLOCK_SIZE_C),
+                        order=(1, 0),
+                    )
+                    tl.store(output_block_ptr, value, boundary_check=(1, 0))
+
+
+@torch.library.custom_op("torch_mlu_triton::fused_slice_low_v2", mutates_args=())
+def fused_slice_low_v2(
```
Comment (zhangjihang-BD) [inline xpu_graph/passes/patterns/targets/mlu/triton_kernel/fused_slice_v2.py:52]: low代表的低维？

File: xpu_graph/passes/patterns/targets/mlu/triton_kernel/fused_slice_v2.py [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    for s in slice_lens:
+        output = torch.empty(
+            (src_tensor.shape[0], s),
+            device=src_tensor.device,
+            dtype=src_tensor.dtype,
+        )
+        outputs.append(output)
+    return outputs
+
+'''
```
Comment (zhangjihang-BD) [inline xpu_graph/passes/patterns/targets/mlu/triton_kernel/fused_slice_v2.py:115]: 删了吧

File: xpu_graph/passes/patterns/targets/mlu/structure_replacements.py [hunk 4]
```diff
@@ -57,11 +61,46 @@ def forward(self, input_tensor, slices):
             16384,  # blocksize
         )
 
+class FuseSliceCatSameInputModule_v2(torch.nn.Module):
+    def forward(self, input_tensor, many_slices):
+        if len(input_tensor.shape) != 2:
+            raise NotImplementedError("input must be 2d")
```
Comment (zhangjihang-BD) [inline xpu_graph/passes/patterns/targets/mlu/structure_replacements.py:67]: 在这里抛异常合理么？前面的pattern似乎没有检查shape？不应该留到这里来做检查吧

File: xpu_graph/passes/patterns/targets/mlu/triton_kernel/fused_slice_v2.py (hunk 2 above)
Comment (jyjyjyjyjyjyjyj) [reply xpu_graph/passes/patterns/targets/mlu/triton_kernel/fused_slice_v2.py:52]: yes

File: xpu_graph/passes/patterns/targets/mlu/structure_replacements.py (hunk 4 above)
Comment (jyjyjyjyjyjyjyj) [reply xpu_graph/passes/patterns/targets/mlu/structure_replacements.py:67]: 这里确实有问题，在这个还没有合入的pr里面修了..https://github.com/BD-Seed-HHW/xpu_graph/pull/105/files
